src/lib.rs

```diff
@@ -214,19 +228,17 @@ impl Log for SimpleLogger {
                 record.module_path().unwrap_or_default()
             };
             #[cfg(feature = "chrono")]
-            {
+            if self.timestamps {
                 println!(
                     "{} {:<5} [{}] {}",
                     Local::now().format("%Y-%m-%d %H:%M:%S,%3f"),
                     level_string,
                     target,
                     record.args()
                 );
+		return;
```

Comment:
out of curiosity: why the `return;` instead of an else?

Reply:
To avoid duplicating the code for the two cases where there is no timestamp.

Reply:
Oh yes, forget the `#[cfg(feature = "chrono")]` just above.